samples/Build2019Demo.Unity/README.md

```diff
@@ -38,22 +38,22 @@ These instructions assume that a HoloLens 2 device will be used to host the shar
 
 ##### HoloLens 2
 
-1. Build UWP Player containing `Demo/Scenes/Finished_Scene.unity` scene, and deploy this application to the HoloLens 2 device.
-2. Launch the `SpectatorView.Build2019Demo` on the HoloLens 2, and wait for the experience to start.
+1. Select `HoloLens` as your target platform using Spectator View's `PlatformSwitcher` (This can be found in the Unity inspector for the `SpectatorView` prefab GameObject).
+2. Build UWP Player containing `Demo/Scenes/Finished_Scene.unity` scene, and deploy this application to the HoloLens 2 device.
```

Comment:
For both this and below, does the language "containing 'XYZ.unity' scene" sufficient enough to describe that it should be in the scene, the first scene, and checked? Is this common enough Unity/instruction knowledge that this is obvious?

Reply:
no, i'll add more images.

---
In reply to: [352940588](https://github.com/microsoft/MixedReality-SpectatorView/pull/290#discussion_r352940588) [](ancestors = 352940588)